src/Adapter/MSTest.Engine/MSTest.Engine.csproj

```diff
@@ -5,15 +5,19 @@
     <RootNamespace>Microsoft.Testing.Framework</RootNamespace>
     <!-- Cancel default license (closed-source) and set license as packaged license file -->
     <PackageLicenseExpressionInternal></PackageLicenseExpressionInternal>
+    <PackageLicenseExpression></PackageLicenseExpression>
     <PackageLicenseFile>License.txt</PackageLicenseFile>
     <!--
       This is a early preview package, keep 1.0.0-alpha or similar suffix even in official builds.
     -->
+    <!-- Whenever VersionPrefix or PreReleaseVersionLabel change, MSTest.Sdk need to be in sync. -->
+    <!-- See GenerateTemplates target in MSTest.Sdk.targets -->
     <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Can we create a property on the main `Directory.Build.props`, like `MSTestEngineVersionPrefix` reuse it here and in the SDK csproj. This would reduce the risk of having them out of sync.